### ISSUE TITLE
Set up webpack

### ISSUE THREAD

**Issue by Anders-E** [Anders-E/Ouija#25]
https://webpack.js.org/
https://webpack.js.org/guides/typescript/

**Comment by Anders-E** [comment]
https://threejs.org/docs/#manual/en/introduction/Import-via-modules

**Comment by Anders-E** [comment]
Work begun in 900a4925fb4549ad70e0c57a637cc311814955c4

Would not work without [three-gltf-loader](https://www.npmjs.com/package/three-gltf-loader) even though it states that:

> starting with three.js r103, GLTFLoader is included in the three package itself and installing three-gltf-loader is no longer necessary.

With three-gltf-loader, graphics output is incorrect ...

![Screenshot from 2019-06-23 00-08-09](https://user-images.githubusercontent.com/11639036/59969327-09bd5680-954b-11e9-83c5-099554e5e1de.png)

... and a lot of warnings are output to console:

```
THREE.WebGLRenderer 105 bundle.js:23070:11
Error: WebGL warning: texImage2D: Requested size at this level is unsupported. bundle.js:20462:19
Error: WebGL warning: renderbufferStorage: Width or height exceeds maximum renderbuffer size. bundle.js:21384:10
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
0.8494504158465224% loaded bundle.js:215:21
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
35.67691746555394% loaded bundle.js:215:21
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
52.66592578248439% loaded bundle.js:215:21
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
66.25713243602874% loaded bundle.js:215:21
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
100% loaded bundle.js:215:21
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
THREE.WebGLRenderer: Texture has been resized from (2940x2048) to (2048x2048). bundle.js:20651:14
THREE.WebGLRenderer: Texture has been resized from (1024x677) to (1024x512). bundle.js:20651:14
THREE.WebGLRenderer: Texture has been resized from (768x1024) to (512x1024).
bundle.js:20651:14
THREE.WebGLRenderer: Texture has been resized from (576x341) to (512x256). bundle.js:20651:14
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0Attachment has no width or height. bundle.js:23555:8
Error: WebGL warning: clear: Framebuffer must be complete. bundle.js:23555:8
Error: WebGL: No further warnings will be reported for this WebGL context. (already reported 32 warnings)
```

**Comment by Anders-E** [comment]
resolved in 02bd0880b46f61c5ce53b409b2def675b9486aba and c17ffb4dbd9dc16ac66b3016ed030d0ff9ee6a89
